src/multiframe.ts

```diff
@@ -67,6 +104,21 @@ const gauss = (x: number, sigma: number): number => {
 const accumBlend = new BlendState(true, BLENDEQUATION_ADD, BLENDMODE_CONSTANT, BLENDMODE_ONE_MINUS_CONSTANT);
 const noBlend = new BlendState(false);
 
+class MyRenderPassShaderQuad extends RenderPassShaderQuad {
```

Comment:
call it something better, `MultiFrameRenderPass`? It shows up in logging.

Reply:
EventfulRenderPassShaderQuad? ;)

Reply:
hehe. Actually, you use it twice for different purpose too. So my suggestion might not be optimal ;)